src/components/NewsletterSignup/NewsletterSignup.tsx

```diff
@@ -0,0 +1,238 @@
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  /** Link to the relevant privacy policy page. */
+  privacyPolicyLink?: string;
+  /** Used to populate the `<h3>` header title. */
+  title?: string;
+  /** The value of the email text input field. */
+  valueEmail?: string;
+  /** Used to specify what is displayed in the component form/feedback area. */
+  view?: NewsletterSignupViewType;
+}
+
+export type NewsletterSignupViewType =
+  | "form"
+  | "submitting"
+  | "confirmation"
+  | "error";
+
+const defaultDescriptionText =
+  "Stay connected with the latest research news from NYPL, including information about our events, programs, " +
+  "exhibitions, and collections.";
+
+/**
+ * The NewsletterSignup component provides a way for patrons to register for an
+ * email-based newsletter distribution list.
+ */
+export const NewsletterSignup = chakra(
+  forwardRef<HTMLDivElement, NewsletterSignupProps>(
+    (
+      {
+        className,
+        confirmationHeading,
+        confirmationText,
+        descriptionText = defaultDescriptionText,
+        errorHeading = "Oops! Something went wrong.",
+        errorText,
+        formHelperText,
+        id,
+        isInvalidEmail = false,
+        newsletterSignupType = "whatsOn",
+        onChange,
+        onSubmit,
+        privacyPolicyLink = "https://www.nypl.org/help/about-nypl/legal-notices/privacy-policy",
+        valueEmail,
+        title = "Sign Up for Our Newsletter!",
+        view = "form",
+        ...rest
+      },
+      ref?
+    ) => {
+      const { isLargerThanMobile } = useNYPLBreakpoints();
+      const styles = useMultiStyleConfig("NewsletterSignup", {
+        newsletterSignupType,
+      });
+      const iconColor = useColorModeValue(null, "dark.ui.typography.body");
+
+      const isFormView = view === "form" || view === "submitting";
+
+      // Manage focus to ensure accessibility when confirmation or error message is rendered.
+      const focusRef = React.useRef<HTMLDivElement>(null);
+      // When view prop changes, set focus to the confirmation or error content element.
+      React.useEffect(() => {
+        focusRef.current?.focus();
+      }, [view]);
+      return (
+        <Stack
+          direction={isLargerThanMobile ? "row" : "column"}
+          ref={ref}
+          __css={styles}
+          {...rest}
+        >
+          <VStack __css={styles.pitch} alignItems="flex-start">
+            {title && <Heading level="h3" text={title} margin="unset" />}
+            {descriptionText ? (
+              typeof descriptionText === "string" ? (
+                <Text margin="unset">{descriptionText}</Text>
+              ) : (
+                descriptionText
+              )
+            ) : null}
+
+            <Link
+              href={privacyPolicyLink}
+              type="external"
+              margin="unset"
+              __css={styles.privacy}
+            >
```

Comment:
I removed the underline, just want to point out that on the figma mock up the privacy policy link is underlined.
As for the `margin="unset`: I removed the ones in the theme file - this was left over from my efforts to move it from the .tsx file to the theme file. I commented the `margin="unset"` out for now but as you will see it will create big spaces around the different text areas of the left side that are not in line with. the figma file.

Reply:
I used the `noSpace` prop.. works excellent! Thank you.